src/integrationTest/sam.test.ts

```diff
@@ -404,6 +408,13 @@ describe('SAM Integration Tests', async function () {
                         testConfig.lambda = {
                             runtime: scenario.runtime,
                         }
+
+                        // little hack for Go, have to set GOPROXY to direct or it will fail to build
+                        // This only applies for our internal systems
+                        if (scenario.language === 'go') {
```

Comment:
nit: This isn't required on codebuild, so not sure if we actually want to do this, or keep a copy that we don't `sam init` (like what we did when there was in an issue in the Python 2 template)

Reply:
Figured it was, but it can be removed if it really isn't needed.

Reply:
Would prefer hacks like this to be kept to a minimum. Since we have a way around it, we should do the workaround instead of the hack in the codebase